hello-books-swagger.json

```diff
@@ -1,97 +1,162 @@
 {
```

Comment:
Great work with the controller @Xwebyna I especially like how you've added an error handling middleware to the application. However, could you please let us know why you deleted the old setup of the `swaggerDoc`?